link-hint.el

```diff
@@ -418,6 +419,26 @@ Only search the range between just after the point and BOUND."
   :open-multiple t
   :copy #'kill-new)
 
+(defun link-hint--treemacs-open ()
+  "Open in treemacs basesed buffer."
+  (if (string-prefix-p " *Treemacs-" (buffer-name))
+      (treemacs-visit-node-in-most-recently-used-window)
+    (treemacs-visit-node-default)))
```

Comment:
When would this condition be false?

Reply:
This is for https://github.com/emacs-lsp/lsp-treemacs . Also, dap-mode will follow with replacing tree-widget with treemacs.